src/renderer/modals/Receive/steps/StepAccount.js

```diff
@@ -116,6 +122,32 @@ export default function StepAccount({
           onChangeToken={onChangeToken}
         />
       ) : null}
+      {account && !receiveTokenMode && tokenTypes.length && account && account.type === "Account" && (
+        <div>
+          <TokenTips
+            textKey="receive.steps.chooseAccount.warningTokenType"
+            textData={{
+              ticker: account.currency.ticker,
+              tokenType: tokenTypes.map(tt => tt.toUpperCase()).join("/"),
+              currency: account.currency.name,
+            }}
+            learnMoreLink={supportLinkByTokenType[tokenTypes[0]]}
+          />
+        </div>
+      )}
+      {account && !receiveTokenMode && tokenTypes.length && account.type === "TokenAccount" && (
+        <div>
```

Comment:
maybe not use a div to group a single component unless this is used for styling issues, avoid it.

Reply:
yes otherwise the mt that is defined in tokentips is not taken into account